example/lib/test_scenarios/bluetooth_state_toggle_scenario.dart

```diff
@@ -30,7 +39,17 @@ class BluetoothStateTestScenario implements TestScenario {
     log("Disabled radio!");
 
     log("Destroying client");
+    await _radioStateSubscription?.cancel();
```

Comment:
Why await?

Reply:
Cancel returns future and I wanted to be sure that this operation will be completed before subscribing for the stream again